docs/user_guide/fairness_in_machine_learning.rst

```diff
@@ -39,6 +39,79 @@ The Fairlearn package is most applicable to two kinds of harms:
   information are extended or withheld. Examples include varying accuracy in
   face recognition, document search, or product recommendation.
 
+Concept Glossary
+----------------------------
+
+The concepts outlined in this glossary are relevant to sociotechnical contexts. 
+
+Construct Validity
+^^^^^^^^^^^^^^^^^^
+
+We explore the concept of construct validity, as described in in Measurement and Fairness [#4]_ , and as it relates to sociotechnical contexts. This is intended to illuminate the concept to readers who are trying to build AI systems in sociotechnical contexts and may also be referenced in explorations that detail how to apply the concept to specific scenarios.
+
+Key Terms 
+~~~~~~~~~
+
+- **Sociotechnical context** – The context surrounding a technical system, including both social aspects (e.g., people, institutions, communities) and technical aspects (e.g., algorithms, technical processes). The sociotechnical context of a system shapes who might benefit or is harmed by AI systems.
+
+- **Unobservable theoretical construct** – An idea or concept that is unobservable and cannot be directly measured but must instead be inferred through observable measurements defined in a measurement model. 
+
+- **Measurement model** – The method and approach used to measure the unobservable theoretical construct.
+
+- **Construct reliability** – This is closely related to construct validity (detailed below) and can be thought of as the extent to which the measurements of an unobservable theoretical construct remain the same when measured at different points in time. A lack of construct reliability can either be due to a misalignment between the understanding of the unobservable theoretical construct and the methods being used to measure that construct, or to changes to the construct itself.
+
+- **Construct validity** – This can be thought of as the extent to which the measurement model measures the intended construct in way that is meaningful and useful.
+
+Key Term Examples  - Unobservable theoretical constructs and Measurement models
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+- **Fairness** is an example of an unobservable theoretical construct. Several measurement models exist for measuring fairness, including demographic parity. See Fairlearn Metrics for more examples of measurement models for measuring fairness. 
```

Comment:
Hmm, I guess I see the term "measurement model" to refer to the process of moving from abstract ideas to concrete, mathematical measurements. And then fairness metrics like demographic parity are one type of measurement, but the modeling process is what gets us (or the readers) from the abstract idea of fairness to that metric.

Reply:
Hmm. I see the metrics in fairlearn.metrics as examples of measurements - and perhaps those measurements may come together to form a bigger 'measurement model'. Maybe I can frame it as: 

**Fairness** is an example of an unobservable theoretical construct. Several measurements exist for measuring fairness, including demographic parity. See Fairlearn Metrics for more examples of measurements for measuring fairness. These measurements may come together to form a measurement model, where several measurements are combined to ultimately measure fairness. 

@hildeweerts @mmadiao 
Does this close the gap?

Reply:
@romanlutz the suggested linking method (:py:mod:fairlearn.metrics) did not work by itself - do I need to define something for this to work?

Reply:
I think I should have phrased this better. I too see the metrics in fairlearn as examples of measurements, i.e., simple measurement models of "fairness". But "fairness" is still an umbrella term that covers all different kinds of things. What I was trying to get to is that fairness metrics can be measurements of different underlying theoretical understanding of what it means for an algorithm to be fair (see section 4 of the paper). How about something like:

Fairness is an example of an unobservable theoretical construct - one about which there have been millennia of disagreements. There are different underlying understandings of what it means for an algorithm to be fair (which can be in conflict with each other!). Simple measurement models of these notions of fairness typically come in the form of fairness metrics, such as demographic parity. See :py:mod:fairlearn.metrics for more examples of fairness metrics.

I think it can be phrased better (the sentences are a bit long and maybe "underlying theoretical understanding"
Feel free to improve the phrasing (I feel the sentences are a bit long) . We might @michaelamoako @mmadaio wdyt?

Reply:
Hmm - so there's a whole section in Measurement and Fairness expanding on the idea of Fairness as an example of an unobservable theoretical construct. In that context, "highly contested" is the phrase used to capture the idea you're getting at. Maybe we should merge the two: 

Fairness is an example of an unobservable theoretical construct that is highly contested: there are many different understandings of what it means for an algorithm to be fair. Simple measurement models of these notions of fairness typically come in the form of fairness metrics, such as demographic parity. See :py:mod:fairlearn.metrics for more examples of fairness metrics.

Reply:
Love it!